deployment/clusterObjectModel/mainParser/layout.py

```diff
@@ -19,38 +19,36 @@
 import logging
 import logging.config
 
-from ..common import linux_shell
+from ...k8sPaiLibrary.maintainlib import common as pai_k8s_common
 
 
+class Layout:
 
-class service_delete:
-
-
-    def __init__(self, service_conf, serivce_name):
-
+    def __init__(self, layout_configuration):
         self.logger = logging.getLogger(__name__)
+        self.layout_configuration = layout_configuration
 
-        self.service_conf = service_conf
-        self.service_name = serivce_name
 
 
+    def validation_pre(self):
+        # TODO
+        return True, None
 
-    def start(self):
 
-        delete_script = "src/{0}/deploy/{1}".format(self.service_name, self.service_conf["delete-script"])
 
-        cmd = "/bin/bash {0}".format(delete_script)
-        err_msg = "Failed to execute the delete script of service {0}".format(self.service_name)
-        self.logger.info("Begin to execute service {0}'s delete script.".format(self.service_name))
-        linux_shell.execute_shell(cmd, err_msg)
+    def validation_post(self, cluster_object_model):
+        return True, None
 
 
 
     def run(self):
+        com_layout = dict()
+        com_layout["machine-sku"] = self.layout_configuration["machine-sku"]
+        com_layout["kubernetes"] = self.layout_configuration["kubernetes"]
+        com_layout["machine-list"] = dict()
 
-        self.start()
-
-
-
+        for host in self.layout_configuration["machine-list"]:
+            com_layout["machine-list"][host["hostname"]] = host
```

Comment:
How to get default password, user accout and ssh port? and other default configuration such docker data?

Reply:
I won't generate password, user account and ssh port.
The quick-start would generate from user's input.

Reply:
For some user like me. I don't like generate configuration. I just want to write my own cluster configuration manaully. Where is the logic to set those configuration to a host?

Reply:
Me too. It's in the layoutCmd.py

Reply:
Sorry, still can't understand this. We need more time to confirm it. It's a very dangerous change for me.

Reply:
You are right, layoutCmd.py is only for aks.
All the others are still in quick-start.

Reply:
But quick-start is for cluster configuraiton generation. And I have no idea how to get this value from the default configuration if user wirte the config manually.